src/wallet/rpc/wallet.cpp

```diff
@@ -610,6 +610,27 @@ RPCHelpMan simulaterawtransaction()
             RPCResult::Type::OBJ, "", "",
             {
                 {RPCResult::Type::STR_AMOUNT, "balance_change", "The wallet balance change (negative means decrease)."},
+                {RPCResult::Type::ARR, "details", "",
+                {
+                    {RPCResult::Type::OBJ, "", "",
+                        {
+                            {RPCResult::Type::STR_HEX, "txid", "the transaction hex of transaction provided by the user"},
+                            {RPCResult::Type::ARR, "wallet_conflicts", "list of txids of transactions in the wallet that conflict with this transaction",
+                            {
+                                {RPCResult::Type::STR_HEX, "txid", "the txid of conflicting transactions in wallet"},
+                            }},
+                            {RPCResult::Type::ARR, "simulated_tx_conflicts", "indices of transactions provided by the user that conflict with this transaction",
+                            {
+                                {RPCResult::Type::NUM, "index", "index of conflicting transactions in list provided by user"},
+                            }},
```

Comment:
Returning txids increases the "manual" work on user's side, if that makes sense? Consider a case when user provides 50 transactions as input. As an end user I think I'd prefer the software telling me "This transaction of yours conflicts with the transaction 11, 17 and 21 that you provided" rather than "This transaction of yours conflict with the transactions with txids <some-long-string1> <some-long-string2> <some-long-string3>"